datasets/acute-care/extract_trio_vcf.py

```diff
@@ -25,7 +25,7 @@ class NotAllSamplesPresent(Exception):
 
 
 def check_samples_in_mt(
-    sample_names: set, family_structures: dict, mat: hl.MatrixTable
+    sample_names: set, family_structures: dict, mat: hail.MatrixTable
```

Comment:
Can we remove the double import of hail at the top, and change this back to `hl.<>`.>?